src/components/FormLayout/FormLayout.tsx

```diff
@@ -27,10 +28,8 @@ export const FormLayout = ({
       onSubmit={onSubmit}
       ref={getRef}
     >
-      <div className={styles['FormLayout__container']}>{children}</div>
-      {Component === 'form' && (
-        <input type="submit" className={styles['FormLayout__submit']} value="" />
-      )}
+      <div>{children}</div>
+      {Component === 'form' && <VisuallyHiddenInput type="submit" value="Отправить" />}
```

Comment:
При `onSubmit === preventDefault` необходимо скрывать этот элемент

```suggestion
      {Component === 'form' && <VisuallyHiddenInput type="submit" value="Отправить" aria-hidden={onSubmit === preventDefault}/>}
```

Reply:
Может, стоит тогда ее убирать для всех, что скажешь?

```suggestion
      {Component === 'form' && onSubmit !== preventDefault && <VisuallyHiddenInput type="submit" value="Отправить" />}
```

Reply:
Убирать не надо, так как `onSumbit` тогда никогда не будет вызван

Reply:
Вынесла эту штуку пока, давай с обсуждением переедем в тот пулл: https://github.com/VKCOM/VKUI/pull/4005